tests/helpers/_resources.py

```diff
@@ -18,3 +22,25 @@ def resolve_resource_path(resource_path: str | Path) -> str:
         The absolute path to the resource.
     """
     return str(_resources_root / resource_path)
+
+
+def check_that_tables_are_close(table1: Table, table2: Table) -> None:
+    """
+    Check that two tables are almost equal.
+
+    Parameters
+    ----------
+    table1: Table
+        The first table.
+    table2: Table
+        The table to compare the first table to.
+    """
+    assert table1.schema == table2.schema
+    for column_name in table1.column_names:
+        assert table1.get_column(column_name).type == table2.get_column(column_name).type
+        assert table1.get_column(column_name).type.is_numeric()
+        assert table2.get_column(column_name).type.is_numeric()
+        for i in range(table1.number_of_rows):
+            entry_1 = table1.get_column(column_name).get_value(i)
+            entry_2 = table2.get_column(column_name).get_value(i)
+            assert (isclose(entry_1, entry_2))
```

Comment:
The `approx` method of `pytest` should yield a better error message if the assertion fails:

https://stackoverflow.com/a/39623614